contrib/README.md

```diff
@@ -0,0 +1,31 @@
+# Contributed CLBOSS Utilities
+
+## Installing
+
+```
+# Install pipx
+sudo apt update
+sudo apt install pipx
```

Comment:
This `apt install pipx` didn't work on my machine.

Perhaps we can just point people to the "official installer"? https://python-poetry.org/docs/#installing-with-the-official-installer

Reply:
Done